### PR TITLE
Enrich filter test case

### PR DESCRIPTION
**Pull request by tydhot**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/context/mock/context.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ package mock
 import (
 	"context"
 	"fmt"
+	"github.com/apache/dubbo-go-pixiu/pkg/model"
```
Comment (AlexStocks) [inline pkg/context/mock/context.go:23]: move it to the third import block. pls.

File: pkg/filter/authority/authority_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package authority
+
+import (
```
Comment (AlexStocks) [inline pkg/filter/authority/authority_test.go:20]: pls split this import block as other code files.

File: pkg/filter/header/header_test.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package header
+
+import (
+	"bytes"
+	"github.com/apache/dubbo-go-pixiu/pkg/context/mock"
```
Comment (AlexStocks) [inline pkg/filter/header/header_test.go:22]: split it.

File: pkg/filter/logger/logger_test.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package logger
+
+import (
+	"bytes"
+	"github.com/apache/dubbo-go-pixiu/pkg/context/mock"
```
Comment (AlexStocks) [inline pkg/filter/logger/logger_test.go:22]: split it.

File: pkg/context/mock/context.go (hunk 1 above)
Comment (tydhot) [reply pkg/context/mock/context.go:23]: ok, I got it

**Comment by tydhot** [comment]
@AlexStocks The imports is spilted now:)

**Review by AlexStocks — APPROVED** [review]

**Review by williamfeng323 — APPROVED** [review]

**Review by xiaoliu10 — APPROVED** [review]

**Pull request merged**